### PR TITLE
JohannaRosenholm -Sofie Rydmark -CarinaMoreno -music release project

### PR DESCRIPTION
**Pull request by RosenholmJohanna**
Link to my deployed project:
https://papaya-fox-74e30c.netlify.app/

### CODE REVIEW

File: code/src/index.css [hunk 1]
```diff
@@ -1,13 +1,259 @@
...
+  text-decoration: none;
+  
+}
+
+header {
+  margin: 10% 10% 0 10%;
+  border-bottom: 0.3px solid #a0a0a0;
+}
+
+a {
```
Comment (uohman) [inline code/src/index.css:24]: A repetition of the code on line 14-15?

File: code/src/components/Playlist.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+const Playlist = ( {playlistName, hrefPlaylist} ) => {
+    return (   
+            <li className="navlinks">
+                <a href={hrefPlaylist} aria-label="Go to playlist" target="_blank">{playlistName}</a>
+            </li>
+            
+        )
+        
+}
+export default Playlist
```
Comment (uohman) [inline code/src/components/Playlist.js:12]: Nice addition with the playlist!

**Review by uohman — COMMENTED** [review]
Not much to comment on. Nice, clean looking code! :)

File: code/src/components/Album.js [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+import AlbumName from './AlbumName'
+import AlbumImage from './AlbumImage'
+
+const Album = ({ title, hrefAlbum, artist, hrefArtist, image }) => {
+    
+    return (
+        <article className="album-card">
+            <AlbumImage image={image} hrefAlbum={hrefAlbum}/>
+            <AlbumName title={title} hrefAlbum={hrefAlbum}/>
+            <ArtistName artist={artist} hrefArtist={hrefArtist}/>
```
Comment (Iracuzzi) [inline code/src/components/Album.js:12]: Maybe could be possible to put all in one file component, and be more easy to find the info about the album.

**Review by Iracuzzi — COMMENTED** [review]
It is a great code, I can't say to much things. Easy to read and understand what did you here. Nice job her!

**Pull request closed without merging**